src/ipyforcegraph/behaviors/shapes.py

```diff
@@ -23,7 +23,10 @@
 
 @W.register
 class Text(HasFillAndStroke):
-    """Draw a text shape, with an optional background."""
+    """Draw a text shape, with an optional background.
+
+    If the ``text`` trait is (or evaluates to) ``0`` or ``None``, no shape will be drawn.
```

Comment:
Would the same happen with ``height``?

Reply:
it would _visually_ hide it, but you'd still pay the canvas price